.circleci/config.yml

```diff
@@ -209,6 +210,31 @@ jobs:
             make -C docs validate-links
       - save_deps_cache
 
+  create-pr-sdk-versions-update:
+    docker:
+      - image: circleci/openjdk:11
+    steps:
+      - github-cli/setup
+      - checkout
+      - restore_deps_cache
+      - setup_sbt
+      - set-sdk-version
+      - run:
+          name: Create PR to update all sdk versions (samples, sbt and maven)
+          command: |
+            chmod +x ./updateSdkReleasedVersion.sh
```

Comment:
not sure why it required this, it should be runable without it I believe.. but CI complained about it when I tested.